v0/destinations/hs/HSTransform-v2.js

```diff
@@ -39,6 +40,24 @@ const {
   getHsSearchId
 } = require("./util");
 
+const RETL_CREATE_ASSOCIATION_OP = "createAssociation";
+const RETL_SOURCE = "rETL";
+
+const addHsAuthentication = (response, destinationConfig) => {
```

Comment:
```suggestion
const addHsAuthentication = (response, Config) => {
```
Generally, in transformer, we go with the `Config` convention. Mentioned just so the readers' have consistency while going through the entire transformer codebase.